source/client-side-operations-timeout/tests/legacy-timeouts.yml

```diff
@@ -1,6 +1,6 @@
 description: "legacy timeouts continue to work if timeoutMS is not set"
 
-schemaVersion: "1.9"
+schemaVersion: "1.0"
```

Comment:
I've noted that some of the CSOT spec tests are generated by template, but this one does not appear to be generated. Something to keep in mind for this, though, as some other specs like CSFLE also use templates.

Reply:
Indeed, the script doesn't take this into account at all.